### ISSUE TITLE
>= 0.204.0 EBUSY errors in cache

### ISSUE THREAD

**Issue by lublak** [n8n-io/n8n#5274]
**Describe the bug**

Since version 0.204.0 there are a lot of problems with EBUSY in the cache (lazy loading implementation).
Based on this: https://community.n8n.io/t/n8n-webhook-empty/22279/9

![grafik](https://user-images.githubusercontent.com/44057030/215117185-91a6448c-c55c-4f05-b6af-304f98023538.png)

```json
{"__type":"$$EventMessageWorkflow","id":"7df56903-962c-47c9-8a77-c5ac9e3955d6","ts":"2023-01-27T15:14:28.515+01:00","eventName":"n8n.workflow.started","message":"n8n.workflow.started","payload":{"executionId":"4","userId":"4f10ffaf-1caf-40e9-9a88-8b692f368904","workflowId":"1","isManual":false,"workflowName":"My workflow"}}
{"__type":"$$EventMessageConfirm","confirm":"7df56903-962c-47c9-8a77-c5ac9e3955d6","ts":"2023-01-27T15:14:28.515+01:00","source":{"id":"0","name":"eventBus"}}
{"__type":"$$EventMessageWorkflow","id":"309ed921-db1f-41f7-a2b6-1e932e265bd6","ts":"2023-01-27T15:14:30.957+01:00","eventName":"n8n.workflow.failed","message":"n8n.workflow.failed","payload":{"executionId":"4","success":false,"userId":"4f10ffaf-1caf-40e9-9a88-8b692f368904","workflowId":"1","errorMessage":"EBUSY: resource busy or locked, copyfile 'C:\\ProgramData\\npm\\npm\\node_modules\\n8n\\node_modules\\n8n-nodes-base\\dist\\nodes\\Google\\Sheet\\googleSheets.svg' -> 'C:\\Users\\testadmin\\.cache\\n8n\\public\\icons\\nodes\\n8n-nodes-base.googleSheets.svg'","isManual":false,"workflowName":"My workflow"}}
{"__type":"$$EventMessageConfirm","confirm":"309ed921-db1f-41f7-a2b6-1e932e265bd6","ts":"2023-01-27T15:14:30.958+01:00","source":{"id":"0","name":"eventBus"}}
```

**To Reproduce**

Install n8n via npm on windows.

**Expected behavior**

No Error

**Environment (please complete the following information):**

- OS: Windows Server 2016
- n8n Version 0.212.1
- Node.js Version v16.15.0
- Database system postgres
- Operation mode own

**Comment by pg-ahenry** [comment]
If i had to guess, This is probably something caused by #4865
More specifically <https://github.com/n8n-io/n8n/blob/9efcf19082413e3a886144e4c178f80f5e24ee27/packages/cli/src/LoadNodesAndCredentials.ts#L412>

**Comment by MajesticPotatoe** [comment]
If i had to guess, This is probably something caused by #4865
More specifically <https://github.com/n8n-io/n8n/blob/9efcf19082413e3a886144e4c178f80f5e24ee27/packages/cli/src/LoadNodesAndCredentials.ts#L412>

**Comment by Joffcom** [comment]
Hey @lublak,

This should be resolved in `0.215.0`, I am going to het this one marked as closed and if you are still seeing the issue we can open it again.

**Comment by lublak** [comment]
@Joffcom thanks :) and i will try this next week.
